tests/functional/spec/components/radio.spec.js

```diff
@@ -0,0 +1,109 @@
+const helpers = require('../../helpers');
+const RadioMandatoryPage = require('../../pages/surveys/radio/radio-mandatory.page');
+const RadioNonMandatoryPage = require('../../pages/surveys/radio/radio-non-mandatory.page');
+const SummaryPage = require('../../pages/surveys/radio/summary.page');
```

Comment:
should call this RadioSummaryPage or take the radio off the other two as they are in the radio folder...for consistency

Reply:
Summary page is a more generic term for all summary pages rather then just the one for radio as it's called summary pages across all spec files

Reply:
I would disagree, the implementation of the summary page for this survey is specific to the survey. we should just rename the constant and not the file `const RadioSummaryPage =`

Reply:
Fineeeeeeeee